versions.yaml

```diff
@@ -18,24 +18,26 @@ versions:
     repo: mongodb3
     tags:
       - 3.4
```

Comment:
I think we should quote the numbers to ensure they are strings. It's OK for a follow up.